code/game/objects/items/fireaxe.dm

```diff
@@ -70,11 +70,25 @@
 	base_icon_state = "bone_axe"
 	name = "bone axe"
 	desc = "A large, vicious axe crafted out of several sharpened bone plates and crudely tied together. Made of monsters, by killing monsters, for killing monsters."
+//SKYRAT EDIT ADDITION BEGIN - Slight antimob axe buff
+	var/faction_bonus_force = 9 //Bonus force dealt against certain factions. Blatantly stolen from the arrow code, because shitcode
```

Comment:
Also this and the variable below need their comment to start with `///` and be on the line before the declaration of the variable, so that it's autodoc-compatible.
Do that on the other places too, I haven't left any comments about it anywhere else.

Reply:
Done iirc.